dbt/adapters/trino/connections.py

```diff
@@ -192,7 +192,8 @@ def open(cls, connection):
             http_headers=credentials.http_headers,
             session_properties=credentials.session_properties,
             auth=auth,
-            isolation_level=IsolationLevel.AUTOCOMMIT
+            isolation_level=IsolationLevel.AUTOCOMMIT,
```

Comment:
Pls do this change on a separate commit.

Reply:
it's a separate commit already